### ISSUE TITLE
Jets DLC Pylons for the Longsword

### ISSUE THREAD

**Issue by Lumnuon** [Scorch052/OPTRE_DEV#94]
It might be a good idea to apply the Jets DLC Pylons feature to the Longsword where applicable. Currently there are no other vehicles that could make use of this feature in the mod, as far as I can tell at least.

**Comment by Scorch052** [comment]
yeah this is a low priority thing but could be worth looking into. Considering White was able to implement the sensor update so well, I trust he'll be able to tackle this as well when the time comes. It may require a complete re-write of a lot of the weapons configs though.

https://community.bistudio.com/wiki/Arma_3_Vehicle_Loadouts

**Comment by BWCSM** [comment]
do we have all of urbans munitions? He said he had made quite a few.
